ynr/apps/results/feeds.py

```diff
@@ -135,7 +135,7 @@ def item_extra_kwargs(self, o):
             "post_id": o.post.slug if o.post else o.old_post_id,
             "winner_person_id": o.winner.id,
             "winner_person_name": o.winner.name,
-            "winner_party_id": o.winner_party.extra.slug,
+            "winner_party_id": o.winner_party.legacy_slug,
```

Comment:
I've make legacy slug unique and populated it in the importer. I reckon that addresses the latter points. For the suggested inline comment – it's used in a few places, do you think here is more important than elsewhere? I think a comment on the model is more helpful?

Reply:
Added help text to the field

Reply:
The reason why I picked out this as a place to leave these comments but not elsewhere is because we are using a legacy slug but assigning to to a key called (`winner_`)`party_id` which is exposed to end-users/the outside world when there is also another identifier in our system which _is actually_ a party_id (or would be better described as that). It seems like more of a potential source of confusion/impact than where it is just an internal variable name.

That said, where we are doing stuff like this:

```py
party_id = kwargs["legacy_slug"]
party = get_object_or_404(Party, legacy_slug=party_id)
```

I do also think that would also be clearer from the perspective of dev/contributor experience if we weren't using the var name `party_id`